src/components/application_manager/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -233,10 +302,28 @@ void CreateInteractionChoiceSetRequest::SendVRAddCommandRequests(
     msg_params[strings::cmd_id] =
         choice_set[strings::choice_set][chs_num][strings::choice_id];
     msg_params[strings::vr_commands] =
-        smart_objects::SmartObject(smart_objects::SmartType_Array);
-    msg_params[strings::vr_commands] =
         choice_set[strings::choice_set][chs_num][strings::vr_commands];
 
+    if (!msg_params[strings::vr_commands].asArray() ||
+        msg_params[strings::vr_commands].empty()) {
+      --expected_chs_count_;
+      SDL_DEBUG("Choice with cmd_id "
+                << msg_params[strings::cmd_id].asString()
+                << "was skipped because it have no vr_commands");
+      continue;
+    }
+
+    if (!InsertVrCommand(vr_commands_hash, msg_params[strings::vr_commands])) {
+      --expected_chs_count_;
+      SDL_DEBUG("Choice with cmd_id "
+                << msg_params[strings::cmd_id].asString()
+                << "was skipped because some of vr_commands already be sent to "
```

Comment:
@LevchenkoS ,please replace text with "was skipped because some of vr_commands were already sent to"